pipelines/managed/fbc-release/fbc-release.yaml

```diff
@@ -382,6 +382,8 @@ spec:
           value: $(params.taskGitUrl)
         - name: taskGitRevision
           value: $(params.taskGitRevision)
+        - name: fbcResultsPath
```

Comment:
I like this param name. The content makes more sense as the actual content is not the internal-request results anymore. Nice one, I will add a ticket to refactor the related tasks.